includes/class-sensei-teacher.php

```diff
@@ -282,6 +282,46 @@ public function teacher_meta_box_content( $post ) {
 
 		<?php
 
+		/**
+		 * Filters the Co-Teachers upgrade link toggle.
+		 *
+		 * @hook   sensei_course_coteachers_hide
+		 * @since  $$next-version$$
+		 *
+		 * @param  {bool} $hide_co_teachers_upgrade_link Whether to hide the Co-Teachers upgrade link.
+		 * @return {bool} Whether to hide the Co-Teachers upgrade link.
+		 */
+		if ( ! apply_filters( 'sensei_course_coteachers_hide', false ) ) {
+			?>
+			<div class="sensei-course-coteachers-wrapper">
+				<?php
+				echo wp_kses(
+					sprintf(
+					// translators: The href tag contains the url to the Sensei Pro landing page.
+						__( 'Multiple teachers? <a href="%s" target="_blank">Upgrade to Pro!</a>', 'sensei-lms' ),
+						'https://senseilms.com/sensei-pro/?utm_source=plugin_sensei&utm_medium=upsell&utm_campaign=co-teachers'
```

Comment:
@burtrw do we want to point the Co-Teachers upgrade link to a specific URL for a landing page and/or redirect? Or should we just leave it as-is (goes to `/sensei-pro`)

Reply:
to /sensei-pro for now. :)